UpdatedSyntheticDataset/SyntheticDataset2/ImageCreator/target_creator.py

```diff
@@ -0,0 +1,211 @@
+        3 = quarter circle with corner facing right
+        4 = quarter circle with corner facing down right
+        5 = quarter circle with corner facing down
+        6 = quarter circle with corner facing down left
+        7 = quarter circle with corner facing left
+        8 = quarter circle with corner facing up left
+
+        9 = half circle with side facing up
+        10 = half circle with side facing right
+        11 = half circle with side facing down
+        12 = half circle with side facing left
+
+        13 = cross with convex facing up
+        14 = cross with concave facing up
+
+        15 = triangle with angle facing up
+        16 = triangle with side facing up
+
+        17 = square with angle facing up
+        18 = square with side facing up
+
+        19 = rectangle with shorter side facing up
+        20 = rectangle with longer side facing up
+
+        21 = trapezoid with shorter side facing up
+        22 = trapezoid with longer side facing up
+
+        23 = pentagon with angle facing up
+        24 = pentagon with side facing up
+
+        25 = hexagon with angle facing up
+        26 = hexagon with side facing up
+
+        27 = heptagon with angle facing up
+        28 = heptagon with side facing up
+
+        29 = octagon with angle facing up
+        30 = octagon with side facing up
+
+        31 = star with convex facing up
+        32 = star with concave facing up
+    """
+
+    @staticmethod
+    def create_specified_single_target(shape_index, letter, size, proportionality, shape_color, letter_color):
+        """
+        :param shape_index: the index of the shape intended to create. The indexes are listed above.
+        :param letter: the letter intended to create.
+        :param size: a parameter of the sizes of the shapes. Different shapes have different ways to use "size"
+                     (for circles, "size" = radius; for squares, "size" = side length, etc.)
+        :param proportionality: a parameter of the proportionality between the shapes and the letters.
+                                "proportionality" is a value, not a specific proportionality between the
+                                dimensions of the shapes and letters as different shapes fit differently with
+                                the letters. 2 is the ideal value of "proportionality". Increasing
+                                "proportionality" decreases the size of the letter while the size of the shape
+                                stays constant.
+        :param shape_color: the intended color of the shape.
+        :param letter_color: the intended color of the letter.
+        :type shape_index: an int from 0 to 32 (inclusive)
+        :type letter: string
+        :type size: an int (representing pixel)
+        :type proportionality: float
+        :type shape_color: (R, G, B, A)
+        :type letter_color: (R, G, B, A)
+        :type R, G, B, and A: int from 0 to 255
```

Comment:
You just need to put these into the above two lines. You shouldn't separate out types like this because if we generate documentation this will cause a problem.